tests/test.js

```diff
@@ -0,0 +1,14 @@
+const myApp = require ('../src/words.js');
+
+const {
+  assert
+} = require('chai').assert;
+
+describe('words', function() {
+  it('returns an object', function() {
+    assert.isObject(myApp.words("A sentence"));
+  });
+  it('returns a correct object', function() {
+    assert.equal(myApp.words("olly olly in come free"),{olly: 2, in: 1, come: 1, free: 1},);
+  });
```

Comment:
Your tests don't pass. Please, check with your colleagues to see how they made their's pass. I will be expecting to see your changes subsequently.